test/browser/time-picker/time-picker-spec.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import TextField from 'TextField';
 import TimePicker from 'TimePicker';
-import DateTime from 'utils/dateTime';
+import {addHours, formatTime} from 'TimePicker/timeUtils';
```

Comment:
@mbrookes @nathanmarks Could any of these tests be moved out of the browser folder?